### ISSUE TITLE
How can I get unet.h5 model

### ISSUE THREAD

**Issue by lbc402** [dattran2346/xray-thesis#5]
When I use the LFS dropdown unet.h5 file, I get the error that "batch response: this repository is over its data quota. Account response for LFS bandwidth should purchase more data packs to restore access."
I would like to further study your code, if you can send unet.h5 model to me by email. The email address is lbc402@126.com. Thank you very much.

**Comment by dattran2346** [comment]
Hi @lbc402, I just upload the model to Kaggle, you can check out [here](https://www.kaggle.com/dattran2346/chestxray14)

**Comment by lbc402** [comment]
Thank to your reply. I'm not sure the link you provided is publicly accessible. In fact, I can't seem to open it.

**Comment by dattran2346** [comment]
Hi @lbc402, I just make it publicly available, you can download the model now.

**Comment by lbc402** [comment]
Sorry @dattran2346 ，When I execute the code in cropped lung.ipynb, the returned _**py**_ variable is a (1,1,256,256) size all 0 matrix. 
`d = {}
distorted_lung = []
for image_name in tqdm(image_names):
    image = Image.open(PATH/IMAGE_DN/image_name).convert('RGB')
    py = unet(V(tfm(image)[None]))   # 扩展第一个维度
    py = (py[0].cpu() > 0.5).type(torch.FloatTensor)`
I confirm that there is no problem with the input image. So I want to confirm with you whether the unet.h5 you provided is correct.

**Comment by lbc402** [comment]
Sorry @dattran2346 ，When I execute the code in cropped lung.ipynb, the returned _**py**_ variable is a zero matrix.
```
d = {}
distorted_lung = []
for image_name in tqdm(image_names):
    image = Image.open(PATH/IMAGE_DN/image_name).convert('RGB')
    py = unet(V(tfm(image)[None]))  
```
I confirm that there is no problem with the input image. So I want to confirm with you whether the unet.h5 you provided is correct.

**Comment by dattran2346** [comment]
@lbc402, did you load the pretrained model? the code to load is [here](https://github.com/dattran2346/chestX-ray-14/blob/9770997601943af18f2fd0d70449bdd9f622aeee/src/unet.py#L62)

The model I uploaded is intended for the web demo, so it is not loaded by default.

**Comment by lbc402** [comment]
I got it. I'm sure the model is loaded. In fact, the chexnet model can run successfully.

**Comment by lbc402** [comment]
The problem is to execute such a piece of code
```
unet = Unet(trained=True, model_name=unet_model).cuda()
unet.eval();
……
image = Image.open(PATH/IMAGE_DN/image_name).convert('RGB')
unet(V(tfm(image)[None])) 
```
And then we get this result
```
tensor([[[[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],
          [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],
          [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],
          ...,
          [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],
          [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],
          [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]]]],
       device='cuda:0', grad_fn=<ReluBackward>)
```

**Comment by dattran2346** [comment]
@lbc402, does the result from unet model are all zeros? Is the `tfm` working correctly, i.e, the image is properly normalized?

**Comment by lbc402** [comment]
@dattran2346 Yes, I got all zero results. I used the initialization parameters you defined, which seems to be no problem. 
```
normalize = transforms.Normalize(IMAGENET_MEAN, IMAGENET_STD)
toTensor = transforms.ToTensor()
tfm = transforms.Compose([
    transforms.Resize((256, 256)),
    toTensor,
    normalize
])
```
I would like to ask you to confirm that this **Unet** model is the best model?

**Comment by dattran2346** [comment]
@lbc402 I only keep the best single model, so yes, it is. But you're getting all zeros, which is quite interesting.

**Comment by dattran2346** [comment]
@lbc402, I just checked the model, it's not all zeros. You should check the min and max value of the mask

Here is the working code
```
image = Image.open('/mnt/DATA/kaggle/siim/images/test/1.2.276.0.7230010.3.1.4.8323329.6249.1517875197.290574.jpg').convert('RGB')
image = Variable(tfm(image)[None])
py = torch.sigmoid(unet(image))
py = (py[0].cpu() > 0.5).type(torch.FloatTensor)
plt.imshow(py.numpy()[0])
```

![download](https://user-images.githubusercontent.com/17774491/69800581-1f286200-1208-11ea-9849-0d1ec5812a92.png)

**Comment by lbc402** [comment]
@dattran2346 Thank you very much. What a naive question I have.
